release-plz.toml

```diff
@@ -0,0 +1,26 @@
+[workspace]
+# disable the changelog for all packages, will only enable for shpool
+changelog_update = false
+
+[changelog]
+header = ""
+body = """
```

Comment:
From the name I suppose we don't want debian style changelog? In that case the release-plz default config works well enough. And since this is a single crate repo, we can just delete the whole config file.

Reply:
Ripped out the file